src/react/features/connectors/effects/useGetRoomEffect.js

```diff
@@ -4,20 +4,22 @@ import { updateConnection } from '../actions'
 import { useConnections } from '../useConnections'
 
 import { getRoom } from '../../diograph/actions'
+import { invokeChannel } from '../../../client/client'
+import { channels } from '../../../../shared/constants'
 
-export const useGetRoomEffect = (getRoomClient, connectorId) => {
+export const useGetRoomEffect = (connectorId) => {
   const { connect, disconnect } = useConnections(connectorId)
 
   const { dispatch, dispatchPromiseAction } = useDispatchActions()
   useEffect(() => {
     connect.forEach(({ address }) => {
       dispatchPromiseAction(
-        () => getRoomClient({ address }),
```

Comment:
This was a generic connector effect independent of folder connector. I would still separate this from folder connector specific methods.

Reply:
I don't understand this connector thing well enough to appreciate having this extra layer and those methods in src/react/features/connectors/folder/client.js

To be honest I was (and still am :D ) pretty proud of myself that I was able to get my head around this thing enough to be able to make it so much simpler!